src/data_scraper/scraper_binance.py

```diff
@@ -20,22 +22,27 @@ def __init__(self, currency_to_buy=config.CURRENCY_TO_BUY, currency_to_sell=conf
         all_currencies.remove(self.currency_to_buy)
         self.currency_pairs = [f'{self.currency_to_buy}{self.currency_to_sell}',
                                *[f'{currency}{self.currency_to_sell}' for currency in all_currencies]]
+        self.col_names = ['Open time', 'Open', 'High', 'Low', 'Close', 'Volume', 'Close time', 'Quote asset volume',
+                          'Number of trades', 'Taker buy base asset volume', 'Taker buy quote asset volume', 'Ignore']
+
+        self.dataset_path = f"{config.FOLDER_TO_SAVE}/binance"
+        self.dataset_name = f"{self.currency_to_buy}{self.currency_to_sell}"
 
-    def get_data(self, start_time, end_time, overwrite=False):
+    def get_data(self, start_time, end_time):
         """Get Binance API exchange rates for all selected currencies. Operates in UTC timezone.
         By default, it will try to continue from the latest available data point if the last saved timestamp is ahead
         of the timestamp specified in the config, to save time instead of loading already available data."""
-        colnames = ['Open time', 'Open', 'High', 'Low', 'Close', 'Volume', 'Close time', 'Quote asset volume',
-                    'Number of trades', 'Taker buy base asset volume', 'Taker buy quote asset volume', 'Ignore']
-        df = pd.DataFrame(columns=colnames)
 
-        for currency_pair in self.currency_pairs:
+        df = pd.DataFrame(columns=self.col_names)
 
+        for currency_pair in self.currency_pairs:
+            start_time_datetime = time_helpers.timestamp_to_str(start_time, format="exact_time")
+            end_time_datetime = time_helpers.timestamp_to_str(end_time, format="exact_time")
             klines = self.client.get_exchange_rates(
                 currency_to_buy=self.currency_to_buy, currency_to_sell=self.currency_to_sell,
-                interval=self.interval, start_time=start_time, end_time=end_time)
+                interval=self.interval, start_time=start_time_datetime, end_time=end_time_datetime)
```

Comment:
why getting data by datetime instead of timestamp? isn't it the same? (actually, timestamp should be even more precise IMO)